src/plugins/data_views/server/fetcher/lib/field_capabilities/field_caps_response.test.js

```diff
@@ -165,5 +165,23 @@ describe('index_patterns/field_capabilities/field_caps_response', () => {
         expect(child).not.toHaveProperty('subType');
       });
     });
+
+    it('sets default field formatter', () => {
+      const fields = readFieldCapsResponse({
+        fields: {
+          percent: {
+            long: {
+              type: 'percent',
+              searchable: true,
+              aggregatable: true,
+              meta: {
+                unit: ['s'],
```

Comment:
I'm a bit confused about this test case. It mixes `percent` and `s` units. Also, wouldn't the type be `long` rather than `percent` or `s`?

Reply:
Good catch, no that doesn't make sense. Fixing, but it has no bearing on the test